test/stdlib/Integers.swift.gyb

```diff
@@ -364,6 +364,21 @@ tests.test("extendingOrTruncating") {
   expectEqual(129, UInt8(extendingOrTruncating: 129 + 1024 as UDWord))
 }
 
+tests.test("Parsing/LosslessStringConvertible") {
+  func _toArray<T: LosslessStringConvertible>(_ text: String) -> [T] {
+    return text.split(separator: " ").map { T(String($0)) }.flatMap { $0 }
+  }
+
+  let i: [Int] = _toArray("1 2 3")
+  expectEqual(i.count, 3)
```

Comment:
FYI: there exists 'expectEqualSequence` https://github.com/apple/swift/blob/master/stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb#L2326.

So one can write checks like these in one shot:

```swift
expectEqualSequence([1, 2, 3], _toArray("1 2 3"))
```